src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -265,19 +285,14 @@ class UploadDetachedFileValidation extends React.Component {
 		validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults}/>;
 		if (!this.state.headerErrors && this.state.validationFinished) {
 			validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} setUploadItem={this.uploadFile.bind(this)} updateItem={this.uploadFile.bind(this)} published={this.state.published}/>;
-			if (this.state.published){
-				// Submission is published and cannot be re-published
+			validationButton = validationButton = <button className='pull-right col-xs-3 us-da-disabled-button' disabled>You do not have permissions to publish</button>;
```

Comment:
It's set to null at the top of the function first, then this set of "if" statements determines what it's set to after that.